roslibrust/Cargo.toml

```diff
@@ -31,6 +31,7 @@ thiserror = "1.0.31"
 [dev-dependencies]
 # Used for logging in examples
 simple_logger = "2.1.0"
+roslibrust_test = { path = "../roslibrust_test" }
```

Comment:
[nit]

I'm starting to be a really big fan of adding comments to Cargo.toml to explain why dependencies exist.
Would generally like a comment here like "contains test message definitions for use in examples and integration tests"